src/SFA.DAS.Commitments.Support.SubSite/Views/Apprenticeships/Index.cshtml

```diff
@@ -0,0 +1,193 @@
+
+                    </tr>
+                    <tr>
+                        <td>Payment status</td>
+                        <td>@Model.PaymentStatus</td>
+
+                    </tr>
+
+                    @if (Model.Alerts != null && Model.Alerts.Any())
+                    {
+                        for (int i = 0; i < Model.Alerts.Count(); i++)
+                        {
+                            if (i == 0)
+                            {
+                                <tr>
+                                    <td>Alerts</td>
+                                    <td><a href="#"> Model.Alerts[i]</a></td>
+
+                                </tr>
+                            }
+
+                            <tr>
+                                <td></td>
+                                <td><a href="#"> Model.Alerts[i]</a></td>
+
+                            </tr>
+                        }
+                    }
+                </tbody>
+            </table>
+
+
+            <table class="responsive twoColumn">
+                <thead>
+                    <tr>
+                        <th colspan="2"> <h2 class="heading-medium">Apprentice</h2></th>
+                    </tr>
+
+                </thead>
+            </table>
+
+            <table class="responsive twoColumn">
+                <tbody>
+                    <tr>
+                        <td>Name</td>
+                        <td>@Model.Name</td>
+
+                    </tr>
+                    <tr id="ULN">
+                        <td>Unique learner number</td>
+                        <td>@Model.ULN</td>
+                    </tr>
+                    <tr>
+                        <td>Date of birth</td>
+                        <td>@Model.DateOfBirth.ToGdsFormatWithSpaceSeperator()</td>
+                    </tr>
+                    <tr id="referenceNumber">
+                        <td>Cohort reference</td>
+                        <td>@Model.CohortReference</td>
+                    </tr>
+
+                    <tr id="referenceNumber">
+                        <td>Employer reference</td>
+                        <td>@Model.EmployerReference</td>
+                    </tr>
+
+                </tbody>
+            </table>
+
+
+            <table class="responsive twoColumn">
+                <thead>
+                    <tr>
+                        <th colspan="2"> <h2 class="heading-medium">Training</h2></th>
+                    </tr>
+                </thead>
+            </table>
+
+            <table class="responsive twoColumn">
+                <tbody>
+                    <tr id="employer">
+                        <td>Legal entity</td>
+                        <td>@Model.LegalEntity</td>
+                    </tr>
+                    <tr id="trainingProvider">
+                        <td>Training provider</td>
+                        <td>@Model.TrainingProvider</td>
+                    </tr>
+                    <tr id="trainingProvider">
```

Comment:
You shouldn't have elements with duplicate id's in html. This looks like a c&p error as the row actually contains UKPRN.
However, as no css or javascript references the ids, we might as well remove them.